apps/vs-code-designer/src/app/commands/createCodeless/createCodelessSteps/CodelessWorkflowCreateStep.ts

```diff
@@ -45,19 +46,68 @@ export class CodelessWorkflowCreateStep extends WorkflowCreateStepBase<IFunction
   }
 
   public async executeCore(context: IFunctionWizardContext): Promise<string> {
+    // Check if the user chose to initialize a static web app
+    if (context.initializeStaticWebApp) {
+      context.telemetry.properties.initializeStaticWebApp = 'true';
+      vscodeExtension.commands.executeCommand('azureLogicAppsStandard.deploy');
+    } else {
+      context.telemetry.properties.initializeStaticWebApp = 'false';
+    }
+
     const template: IWorkflowTemplate = nonNullProp(context, 'functionTemplate');
     const functionPath: string = path.join(context.projectPath, nonNullProp(context, 'functionName'));
-
-    const emptyStatefulDefinition: StandardApp = {
-      definition: {
-        $schema: 'https://schema.management.azure.com/providers/Microsoft.Logic/schemas/2016-06-01/workflowdefinition.json#',
-        actions: {},
-        contentVersion: '1.0.0.0',
-        outputs: {},
-        triggers: {},
-      },
-      kind: 'Stateful',
-    };
+    let emptyStatefulDefinition: StandardApp;
+    if (context.initializeStaticWebApp) {
+      //initiate basic logic app which can be called from SWA
+      emptyStatefulDefinition = {
```

Comment:
should we read this from a file instead?

Reply:
we have a file to store the templates apps/vs-code-designer/src/app/utils/codeless/templates.ts